core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/ChangeLoggable.java

```diff
@@ -10,8 +11,15 @@
  */
 public interface ChangeLoggable {
 
+  public static final Pattern COMMA = Pattern.compile(",");
+
   public Collection<ChangeLog> getChangeLog();
 
   public ChangeLog createChangeLog(String summary, String columnsChanged, User user);
 
+  public default boolean didSomeoneElseChangeColumn(String referenceColumnName, User nonHumanUser) {
+    return getChangeLog().stream().filter(cl -> cl.getUser().getUserId() != nonHumanUser.getUserId())
```

Comment:
This means the lazy-loaded changelogs will be initialized for every notification processed. Might not matter, but since this is only useful for runs and only for two fields, would it be better to just include a boolean field for each to mark not-overrideable-by-runscanner?